cmd/containerd/main.go

```diff
@@ -254,3 +274,17 @@ func getNATSPublisher(eventsAddress string) (*nats.EncodedConn, error) {
 
 	return nec, nil
 }
+
+func expvarHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json; charset=utf-8")
+	fmt.Fprintf(w, "{\n")
+	first := true
+	expvar.Do(func(kv expvar.KeyValue) {
```

Comment:
would not it be better to build a map[string]string and use a built-in JSON encoder to do the rest of the job? it can handle escaping and other JSON specific standards for you.